test/libsolidity/SemanticTest.cpp

```diff
@@ -164,6 +165,36 @@ map<string, Builtin> SemanticTest::makeBuiltins()
 				return toBigEndian(h256(ExecutionFramework::setAccount(accountNumber).asBytes(), h256::AlignRight));
 			}
 		},
+		{
+			"bytecode",
+			[this](FunctionCall const& _call) -> optional<bytes>
+			{
+				bool result{false};
+				optional<regex> regexPattern;
+				soltestAssert(_call.arguments.parameters.size() == 1, "Bytecode pattern string expected.");
+				string pattern{_call.arguments.parameters.at(0).rawString};
+				bytes bytecode = ExecutionFramework::bytecode();
+				if (pattern.length() >= 2 && boost::starts_with(pattern, "\"") && boost::ends_with(pattern, "\""))
+				{
+					pattern = pattern.substr(1, pattern.length() - 2);
+					try
+					{
+						regex expr(pattern);
+						regexPattern = expr;
+					}
+					catch(...)
+					{
+						regexPattern.reset();
```

Comment:
```suggestion
						soltestAssert(false, "Must be a valid regexp if enclosed in quotes.");
```

Reply:
Or we could just not catch the exception in the first place :)

Though having a more specific message is not bad either. It would be best if there was some kind of exception chaining mechanism so that we could have both visible when they bubble up to the top.

Reply:
That is also an option, or catching it and displaying it with `soltestAssert(false, e.message())`?